site/pages/home/newsletter-slice/index.js

```diff
@@ -0,0 +1,24 @@
+// import classnames from 'classnames/bind';
```

Comment:
Can we maybe have Newsletter Slice as a separate change, as it doesn't seems to be part of this PR